Contributing Blink's Web Storage layout tests

chunywang opened the pull request:
Original layout tests: https://chromium.googlesource.com/chromium/blink

chunywang committed:
Contributing Blink's Web Storage layout tests

chunywang commented:
I will rewrite those tests under testharness.js.

sideshowbarker commented:
@chunywang please also make sure to remove trailing whitespace; see https://travis-ci.org/w3c/web-platform-tests/builds/63601253

chunywang commented:
I have rewritten blink's web storage test under testharness.js. Call for review.

zqzhang commented:
@Ms2ger are you available for reviewing the rest in critic?

chunywang committed:
Update Blink's Web Storage layout tests under testharness.js

- Update tests under testharness.js
- Remove some blink's tests as they are covered by
  https://github.com/w3c/web-platform-tests/blob/master/webstorage/storage_clear.html
  https://github.com/w3c/web-platform-tests/blob/master/webstorage/missing_arguments.html
  https://github.com/w3c/web-platform-tests/blob/master/webstorage/storage_removeitem.html
  https://github.com/w3c/web-platform-tests/blob/master/webstorage/storage_setitem.html
